index.html

```diff
@@ -6673,9 +6718,11 @@ <h3 id="Elemental">Elemental</h3>
                <td>Floating Stone Shrapnel
                </td>
                <td>
-                  Create a 15ft radius area around you that moves with you. Every 5ft any 
-                  creature is moved as a part of a push or pull through this area it suffers Light 
-                  damage.
+                  <strong>Shards of stone circle around you, harming all foes forced through it.</strong>
+                  <p>
+                     Create a 15ft radius area around you that moves with you. For every 5ft any 
+                     creature is moved, as a part of a push or pull movement, through this area it suffers Light 
```

Comment:
comma splice here is unnecessary and makes parsing the sentence harder

Reply:
Also maybe and clarity that this persists to the start of your next turn?  I know most do, but this one leaves a bit of a question

Reply:
Done, idk about stating that it only lasts a turn? Why is this different to any other ability? The same assumption holds